### ISSUE TITLE
Additional Status Definitions

### ISSUE THREAD

**Issue by cbarnardne** [MicrosoftDocs/OfficeDocs-Enterprise#147]
I have noticed that some Incidents have a status of "False Positive" or "Information Unavailable", It would be nice to have these additional status types in this documentation along with how Microsoft defines them.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dd2a834a-296b-1c7d-7358-6f6a19f8f3c3
* Version Independent ID: 9ab6adbe-c651-d833-f395-64b7cddb2dd2
* Content: [How to check Office 365 service health](https://docs.microsoft.com/en-us/office365/enterprise/view-service-health?redirectSourcePath=%252fen-us%252farticle%252fHow-to-check-Office-365-service-health-932AD3AD-533C-418A-B938-6E44E8BC33B0#PickTab=Definitions)
* Content Source: [Enterprise/view-service-health.md](https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/blob/live/Enterprise/view-service-health.md)
* Service: **o365-administration**
* GitHub Login: @kfollis
* Microsoft Alias: **kfollis**

**Comment by denisebmsft** [comment]
@cbarnardne thank you for the feedback. We are looking into this.

**Comment by AndreaBarr** [comment]
@cbarnardne Thank you for this feedback

I agree it would be appropriate and convenient to have those added to this article. 

I was able to get this into the backlog for the Enterprise team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.

**Comment by robmazz** [comment]
I've updated this topic to include the "Post-incident report published" status and definition, as well as updating the topic for some formatting issues. After discussing with the product group team, we've decided not to include status/definition information on "False Positive" and "Information Unavailable" - these are typically edge cases and will not be normally displayed to customers. These are temporary statuses used for internal tracking.
